caom2pipe/astro_composable.py

```diff
@@ -163,6 +165,37 @@ def check_fits(fqn):
     return True
 
 
+def check_fitsverify(fqn):
+    """
+    Execute fitsverify on fqn
+    :return: bool True if compliant, False if Error count > 0
+    """
+    result = True
+    if '.fits' in fqn:
```

Comment:
The `fitsverify` command will spit back a `verify: FAILED` error if the file is not a `FITS` file anyway, so this can be removed unless you really want to shortcut it for readability, or to send back a custom error.

Reply:
Avoid the `subprocess.Popen` call, as it's mildly expensive.